tools/vscode-azurewebpubsub/src/constants.ts

```diff
@@ -84,6 +85,18 @@ export const MINUTE_MILLISECONDS = 60 * 1000;
 export const HOUR_MILLISECONDS = 60 * MINUTE_MILLISECONDS;
 export const DAY_MILLISECONDS = 24 * HOUR_MILLISECONDS;
 
+export const LOCAL_TUNNEL_NODE_PACKAGE_NAME = "@azure/web-pubsub-tunnel-tool";
+export const LOCAL_TUNNEL_EVENT_HANDLER: EventHandler = {
+    urlTemplate: "tunnel:///eventhandler",
+    userEventPattern: "*",
+    systemEvents: ["connected", "disconnected"]
+}
+export const LOCAL_TUNNEL_EXISTENCE_STATUS_KEY = "localTunnelExistenceStatus";
+export const LOCAL_TUNNEL_UPSTREAM_SERVER_URL = "http://localhost:3000";
```

Comment:
could we support specifying the port in vscode? It might be common since vscode is where to write the upstream server code.

Reply:
updated